circle.yml

```diff
@@ -3,23 +3,26 @@ machine:
     - docker
   environment:
     CLOUDSDK_CORE_DISABLE_PROMPTS: 1
-    image_name: codeclimate-rubocop
-
-dependencies:
-  pre:
-    - echo $gcloud_json_key_base64 | sed 's/ //g' | base64 -d > /tmp/gcloud_key.json
-    - curl https://sdk.cloud.google.com | bash
-    - gcloud auth activate-service-account $gcloud_account_email --key-file /tmp/gcloud_key.json
-    - gcloud docker -a
+    PRIVATE_REGISTRY: us.gcr.io/code_climate
+    PUBLIC_REGISTRY: hub.docker.com/codeclimate
 
 test:
   override:
     - bundle exec rake
-    - docker build -t=$registry_root/$image_name:b$CIRCLE_BUILD_NUM .
+    - docker build -t=$PRIVATE_REGISTRY/$CIRCLE_PROJECT_REPONAME:b$CIRCLE_BUILD_NUM .
 
 deployment:
   registry:
     branch: master
+    owner: codeclimate
     commands:
-      - docker push $registry_root/$image_name:b$CIRCLE_BUILD_NUM
-
+      - echo $GCLOUD_JSON_KEY_BASE64 | sed 's/ //g' | base64 -d > /tmp/gcloud_key.json
+      - curl https://sdk.cloud.google.com | bash
+      - gcloud auth activate-service-account $GCLOUD_ACCOUNT_EMAIL --key-file /tmp/gcloud_key.json
```

Comment:
Oh sure. That's not how I remembered it, but if it works without it then even better